README.md

```diff
@@ -36,7 +36,7 @@ python binding documentation:
 
 ```
 >>> import pymanifold
->>> help(manifold)
+>>> help(pymanifold)
```

Comment:
This reminds me; our library is just called `manifold`, the JS is `manifold.js`. What do you say we just call the python bindings `manifold` as well?

Reply:
The name manifold was taken by another package in pypi, so we need another name.

Reply:
Bah! Hasn't even been updated since 2009; wish there was a way to take over abandoned names. Oh well.